Realm/Realm/Handles/AppHandle.cs

```diff
@@ -205,6 +213,22 @@ public static AppHandle CreateApp(Native.AppConfiguration config, byte[] encrypt
             return new AppHandle(handle);
         }
 
+        public static void ForceCloseHandles()
+        {
+            lock (_appHandles)
+            {
+                foreach (var weakHandle in _appHandles)
+                {
+                    if (weakHandle.Target is AppHandle handle)
```

Comment:
Are we only being overly cautious here with the is operator and then then the null conditional operator? It seems it can't be anything else than AppHandle